pallets/interest-accrual/src/lib.rs

```diff
@@ -95,6 +95,14 @@ pub struct RateDetails<InterestRate> {
 	pub reference_count: u32,
 }
 
+#[derive(Encode, Decode, Default, TypeInfo, PartialEq)]
+#[repr(u32)]
+pub enum Release {
+	#[default]
+	V0,
+	V1,
+}
+
```

Comment:
Not blocking, but WDYT about moving this to common-types in the future and if somebody has a new version of its storage, we simply create another variant?

Reply:
I think it's better to track the allowed versions for each crate separately, just to avoid any potential for accidentally putting the wrong versions into storage